### ISSUE TITLE
Installation with partition-level encryption is left unencrypted

### ISSUE THREAD

**Issue by RealOrangeOne** [Antergos/Cnchi#741]
### Problem:
Installing Antergos with a manually encrypted partition doesnt actually encrypt the partition. This feels like a _really_ important issue!

### Steps To Reproduce:
1. Run Cnchi, Use custom partitioning
2. Mount points for swap and /boot as normal
3. Specify `/` on EXT4 partition, enable encryption, specify password
4. Continue install (on review, clearly says _"... and encrypted."_)
5. GParted shows correct partitions and sizes, but root is unencrypted

### Log Files
- `cnchi.log`: https://ghostbin.com/paste/jookk

### Additional Notes:
Repeatable on both a virtual machine, and laptop itself.

Can't use full-disk encryption, as I also have a windows install on the same disk.

**Comment by lots0logs** [comment]
@karasu Any idea?

**Comment by karasu** [comment]
Nope, will check it asap

**Comment by karasu** [comment]
@RealOrangeOne 
Did you set a volume name and a password  to the encrypted partition?

**Comment by RealOrangeOne** [comment]
@karasu yes to volume name, no to partition label. I can try setting a partition label and see if that makes a difference?

**Comment by karasu** [comment]
@RealOrangeOne I don't think it will make any difference. Give me some time to do more tests...
